articles/changes.md

```diff
@@ -0,0 +1,241 @@
+---
+layout: default
+title: Changes between ROS 1 and ROS 2
+permalink: articles/changes.html
+abstract:
+  This article provides an overview about the changes being made in ROS 2 compared to ROS 1.
+published: true
+author: '[Dirk Thomas](https://github.com/dirk-thomas)'
+---
+
+* This will become a table of contents (this text will be scraped).
+{:toc}
+
+# {{ page.title }}
+
+<div class="abstract" markdown="1">
+{{ page.abstract }}
+</div>
+
+Original Author: {{ page.author }}
+
+
+## Preface
+
+Each change is described as briefly as possible but giving enough context and rational for a reader familiar with ROS 1.
+
+
+## Platforms and dependencies
+
+### Platforms
+
+ROS 1 is only being CI tested on Ubuntu.
+It is actively supported by the community on other Linux flavors as well as OS X.
+
+ROS 2 will be CI tested and supported on Ubuntu, OS X as well as Windows.
+
+
+### Languages
+
+#### C++ standard
+
+ROS 1 is targeting C++03.
+ROS 2 will use C++11.
+
+
+#### Python
+
+ROS 1 is targeting Python 2.
+ROS 2 will use 3.4+.
+
+
+### Reusing existing middleware
+
+ROS 1 uses a custom serialization format, a custom transport protocol as well as a custom central discovery mechanism.
+ROS 2 will use an abstract interface and provide multiple implementations of that interface based using existing DDS implementations.
```

Comment:
This sentence is confusing, maybe `ROS 2 will have an abstract middleware interface, through which serialization, transport, and discovery will be provided by one of several existing DDS or RTPS implementations.`?